on-screen-keyboard-input/src/App.tsx

```diff
@@ -1,13 +1,18 @@
-import { FC } from 'react'
```

Comment:
I removed FC type throughout a project, since it's not needed anymore, I even thought it was deprecated already
https://github.com/facebook/create-react-app/pull/8177